artifacts/CAMARA_common.yaml

```diff
@@ -492,25 +492,26 @@ components:
               value:
                 status: 409
                 code: ABORTED
-                message: Concurrency conflict.
+                message: "Resource is being modified by another operation. Please retry."

```

Comment:
Would take off quotation marks in message as per alignment with the rest of Common erros. Same in other exceptions updated